util/dvsim/FlowCfg.py

```diff
@@ -402,9 +417,12 @@ def _create_deploy_objects(self):
     def create_deploy_objects(self):
         '''Public facing API for _create_deploy_objects().
         '''
+        self.prune_selected_cfgs()
         if self.is_master_cfg:
             self.deploy = []
             for item in self.cfgs:
+                log.info("[scratch_dir]: [%s]: [%s]", item.name,
```

Comment:
Looks like this will no longer be printed if you pass a non-master cfg. See if there is a better place to put it.